src/compute/src/render/threshold.rs

```diff
@@ -96,21 +96,29 @@ where
     let arrangement = input
         .arrangement(&key)
         .expect("Arrangement ensured to exist");
-    let negatives = match &arrangement {
-        ArrangementFlavor::Local(oks, _) => {
-            threshold_arrangement(oks, "Threshold retractions local", |count| *count < 0)
+    let (oks, negatives, errs) = match &arrangement {
```

Comment:
This entire block also appears untested and quite interesting (this actually seems suspicious to me, if there should be a test for this I can check if coverage is broken in some way)

Reply:
Yep, the whole function cannot be exercised through SQL because the optimizer never generates a plan for it. Although it'd be nice to actually use it, it seems we're better off removing it, and bringing it back once we actually can use it. I believe the implementation has a bug that this change fixes.

Reply:
Dead code should have been removed.